src/go/gcsrunner/main/runner.go

```diff
@@ -54,6 +54,7 @@ var (
 	envoyBinaryPath = flag.String("envoy_bin_path", "bin/envoy", "Location of the Envoy binary.")
 	envoyLogLevel   = flag.String("envoy_log_level", "info",
 		"Envoy logging level. Default is `info`. Options are: [trace][debug][info][warning][error][critical][off]")
+	overridePlatform = flag.String("override_platform", "", "If specified, configures `platform` field in Service Control reports.")
```

Comment:
There is "ComputePlatformOverride" already in the options. 

I would think you only need to set this in the src/go/bootstrap/static/bootstrap.go, set it to "Cloud API Gateway".